roadmap.md

```diff
@@ -0,0 +1,38 @@
+# Usage of Epics
+An Epic is a grouping of smaller related issues/tasks that fits into a larger goal. An Epic can potentially span across multiple iterations. 
+
+To track how smaller issues fit into the larger goal, we use [task lists](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists).
+Breaking a large issue into smaller issues makes the work more manageable and enables team members to work in parallel. 
+It also leads to smaller pull requests, which are easier to review.
+
+>NOTE: To create an epic on Github, use the available Epic template.
+
+# Usage of Github issue template/forms
+We have [customized templates](https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/blob/main/.github/ISSUE_TEMPLATE/epic.md) that helps contributors in 
+creating new issues in the repository. When creating an issue, choose the template that best fits the issue you are about to create.  
+
+# Communicate
+- Use `@mentions` to alert a person or entire team about a the issue. 
```

Comment:
Should we have a triage process for different types of issues? How would a creator know which is the correct person to alert about a new issue or Epic?

Reply:
I was hoping we could use teams for this. And then we'll have a specific team that part of the triaging process. Once that happens we'll change the `@mentions` to `@ffm/triage`

Reply:
That makes a ton of sense. Let's tease that out now as well.

Since ffm will be used by multiple teams e.g. Quarkus and Golang fleet managers. Should there be a single triage team or separate based on the related project?

Reply:
> I was hoping we could use teams for this. And then we'll have a specific team that part of the triaging process. Once that happens we'll change the @mentions to @ffm/triage

Will it make sense to use the [factorized-fleet](https://github.com/bf2fc6cc711aee1a0c2a/org-management/blob/main/org/config.yaml#L947) team for now and change `@mentions` with that team?

Reply:
Thanks.